components/tidb_query_vec_expr/src/impl_compare.rs

```diff
@@ -239,6 +242,32 @@ pub fn greatest_real(args: &[&Option<Real>]) -> Result<Option<Real>> {
     do_get_extremum(args, |x, y| x.max(y))
 }
 
+#[rpn_fn(varg, min_args = 2, capture = [ctx])]
+#[inline]
+pub fn greatest_time(ctx: &mut EvalContext, args: &[&Option<Bytes>]) -> Result<Option<Bytes>> {
+    let mut greatest = None;
+    for arg in args {
+        match arg {
+            Some(arg_val) => {
+                let s = from_utf8(arg_val).unwrap();
```

Comment:
Prefer to import only `std::str` and use `str::from_utf8`.
And if the input is invalid utf8, we should return an error instead of panic here.